components/Player/PlayerTabs.jsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import TabBar from '../TabBar';
 import { playerPages } from '../Header/Pages';
 
-const playerPagesMapped = (accountId) => playerPages.map((e) => Object.assign({}, e, {
-  route: `/players/${accountId}/${e.name.toLowerCase()}`,
-  label: e.name,
+const playerPagesMapped = (accountId) => playerPages.map(page => ({
+  route: `/players/${accountId}/${page.name.toLowerCase()}`,
```

Comment:
It should be fine to merge this since it's valid in CI.  However if you want your local syntax highlighting to work, you'll either have to figure out your IDE (probably a spread operator plugin), or just go back to Object.assign()